packages/edit-site/src/components/editor/index.js

```diff
@@ -48,7 +48,7 @@ const interfaceLabels = {
 	drawer: __( 'Navigation Sidebar' ),
 };
 
-function Editor() {
+function Editor( { initialSettings } ) {
```

Comment:
I would appreciate a sanity check from someone familiar with the edit site's editor initialization.